### ISSUE TITLE
Bitbucket Oauth 1.0 doesn't work anymore?

### ISSUE THREAD

**Issue by teolaz** [YahnisElsts/plugin-update-checker#310]
Hi @YahnisElsts, i continue to have 401 errors from Bitbucket even after the 4.7 update... 
my repo is private and i created an oauth consumer with the same method that worked before 2018 ("this repo is private" etc etc).
![image](https://user-images.githubusercontent.com/19909323/64949788-4236ad00-d87a-11e9-80bd-d6bd83ecb468.png)
Is it possible they're moving the authorization to Oauth 2 abandoning completely the old Oauth 1? See more here: https://developer.atlassian.com/bitbucket/api/2/reference/meta/authentication

**Comment by YahnisElsts** [comment]
For what it's worth, it still works for me. I tested it today. I also tried adding a new OAuth consumer to check if BitBucket is doing some backwards-compatibility magic to make my old consumer credentials work, but the new consumer also works fine.

![image](https://user-images.githubusercontent.com/2527434/64954602-cabf4880-d88e-11e9-9eed-754cf5b25bb6.png)

This may sound obvious, but have you enabled the "Repositories" -> "Read" permission for your OAuth consumer?
